pkg/remotedialer/wsconn.go

```diff
@@ -24,7 +24,6 @@ func newWSConn(conn *websocket.Conn) *wsConn {
 func (w *wsConn) WriteMessage(messageType int, data []byte) error {
 	w.Lock()
 	defer w.Unlock()
-	w.conn.SetWriteDeadline(time.Now().Add(PingWaitDuration))
```

Comment:
What is the intention of removing the write deadline?  Not setting a deadline on write leads to hung connections, blocked goroutines, and memory leaks.